src/lib/state.ts

```diff
@@ -50,6 +51,16 @@ export const dumpValue = (
   return valueState;
 };
 
+export const dumpValueNotification = (
+  node: ZWaveNode,
+  valueArgs: ZWaveNodeValueNotificationArgs
+): ValueState => {
+  const valueState = valueArgs as ValueState;
+  valueState.metadata = node.getValueMetadata(valueArgs);
+  valueState.ccVersion = 0; // no need for ccVersion in these stateless events (for now)
```

Comment:
If there is no need, let's not include it?

Reply:
Decided to keep it in for consistency reasons.